gigamon/assets/logs/gigamon_tests.yaml

```diff
@@ -0,0 +1,138 @@
+        "dns_qdcount": "1",
+        "dns_arcount": "2",
+        "dns_transaction_id": "18011",
+        "dns_name": "vcsa.vmware.com.cdn.cloudflare.net",
+        "dns_host": "vcsa.vmware.com.cdn.cloudflare.net",
+        "dns_host_addr": "162.159.140.167",
+        "dns_host_type": "OPT",
+        "dns_reply_code": "0",
+        "dns_response_time": "0.000561",
+        "dns_ttl": "81",
+        "dns_flags": "256",
+        "dns_opcode": "0",
+        "dns_class": "1",
+        "dns_host_class": "1",
+        "dns_host_raw": "766373612e766d776172652e636f6d2e63646e2e636c6f7564666c6172652e6e6574",
+        "dns_query": "vcsa.vmware.com",
+        "dns_query_type": "1",
+        "app_id": "32",
+        "ip_version": "4",
+        "src_bytes": "86",
+        "dst_bytes": "166",
+        "dst_packets": "1",
+        "start_time": "2024:08:29 2:14:38.861",
+        "end_time": "2024:08:29 12:14:38.862",
+        "intf_name": "0",
+        "egress_intf_id": "0",
+        "sys_up_time_first": "354805",
+        "sys_up_time_last": "354806",
+        "end_reason": "2",
+        "app_name": "dns",
+        "id": "3844000450554626049",
+        "seq_num": "18085532",
+        "ddsource": "gigamon" 
+               
+      }
+    result: null
+  - sample: 
+      {
+        "ts": "Thu Aug 29 12:15:37 2024",
+        "vendor": "Gigamon",
+        "version": "6.6.00",
+        "generator": "gs_apps_appInst9_423330e5-4a3d-f203-14cb-b6642e5054e9",
+        "dst_mac": "00:00:0c:9f:f7:f8",
+        "src_mac": "00:50:56:b3:a4:e1",
+        "src_ip": "10.203.43.1",
+        "dst_ip": "91.189.91.49",
+        "protocol": "6",
+        "src_port": "55856",
+        "dst_port": "80",
+        "device_inbound_interface": "0",
+        "http_server": "connectivity-check.ubuntu.com",
+        "http_uri": "\\/",
+        "http_uri_full": "\\/",
+        "http_server_agent": "nginx\\/1.14.0 (Ubuntu)",
+        "http_code": "204",
+        "http_uri_path": "\\/",
+        "http_request_size": "87",
+        "http_host": "connectivity-check.ubuntu.com",
+        "http_uri_decoded": "\\/",
+        "http_uri_path_decoded": "\\/",
+        "http_user_agent":"Mozilla\\/5.0 (Windows; U; Windows NT 6.1; en-US) AppleWebKit\\/534. 10 (KHTML, like Gecko) Chrome\\/8.0.552.237 Safari\\/534.10",
+        "http_uri_raw": "\\/",
+        "http_method": "GET",
+        "http_version": "1.1",
+        "app_id": "3006",
+        "ip_version": "4",
+        "src_bytes": "491",
+        "dst_bytes": "461",
+        "src_packets": "6",
+        "dst_packets": "4",
+        "start_time": "2024:08:29 12:15:36.202",
+        "end_time": "2024:08:29 12:15:36.997",
+        "intf_name": "0",
+        "egress_intf_id": "0",
+        "sys_up_time_first": "412146",
+        "sys_up_time_last": "412941",
+        "end_reason": "3",
+        "app_name": "ubuntu",
+        "id": "3844000451458498561",
+        "seq_num": "18085527",
+        "ddsource": "gigamon"
+      }
+
+    result: null
```

Comment:
```suggestion
id: gigamon
tests:
  - sample: |-
      {
        "ts": "Thu Aug 29 10:49:40 2024",
        "vendor": "Gigamon",
        "version": "6.6.00",
        "generator": "gs_apps_appInst9_423330e5-4a3d-f203-14cb-b6642e5054e9",
        "dst_mac": "ec:2a:72:22:8a:fa",
        "src_mac": "00:0c:29:3d:5c:e6",
        "src_ip": "10.203.41.4",
        "dst_ip": "10.203.41.158",
        "protocol": "6",
        "src_port": "55861",
        "dst_port": "9080",
        "device_inbound_interface": "0",
        "ssl_common_name": "10.203.41.158",
        "ssl_issuer": "CA",
        "ssl_validity_not_before": "2024-05-26 13:01:03",
        "ssl_validity_not_after": "2029-05-2613:01:03",
        "ssl_cipher_suite_id": "49200",
        "app_id": "4962",
        "ip_version": "4",
        "src_bytes": "2728",
        "dst_bytes": "5045",
        "src_packets": "13",
        "dst_packets": "11",
        "start_time": "2024:08:29 10:49:40.710",
        "end_time": "2024:08:29 10:49:40.735",
        "intf_name": "0",
        "egress_intf_id": "0",
        "sys_up_time_first": "36697",
        "sys_up_time_last": "36722",
        "end_reason": "3",
        "app_name": "Unknown ssl",
        "id": "3844000378249019393",
        "seq_num": "18045114",
        "ddsource": "gigamon"
      }
    result: null
  - sample: >-
      {
        "ts": "Thu Aug 29 12:15:39 2024",
        "vendor": "Gigamon",
        "version": "6.6.00",
        "generator": "gs_apps_appInst9_423330e5-4a3d-f203-14cb-b6642e5054e9",
        "dst_mac": "00:00:0c:9f:f7:f8",
        "src_mac": "00:0c:29:fc:7c:fc",
        "src_ip": "10.203.41.3",
        "dst_ip": "10.200.50.11",
        "protocol": "17",
        "src_port": "33975",
        "dst_port": "53",
        "device_inbound_interface": "0",
        "dns_qdcount": "1",
        "dns_arcount": "2",
        "dns_transaction_id": "18011",
        "dns_name": "vcsa.vmware.com.cdn.cloudflare.net",
        "dns_host": "vcsa.vmware.com.cdn.cloudflare.net",
        "dns_host_addr": "162.159.140.167",
        "dns_host_type": "OPT",
        "dns_reply_code": "0",
        "dns_response_time": "0.000561",
        "dns_ttl": "81",
        "dns_flags": "256",
        "dns_opcode": "0",
        "dns_class": "1",
        "dns_host_class": "1",
        "dns_host_raw": "766373612e766d776172652e636f6d2e63646e2e636c6f7564666c6172652e6e6574",
        "dns_query": "vcsa.vmware.com",
        "dns_query_type": "1",
        "app_id": "32",
        "ip_version": "4",
        "src_bytes": "86",
        "dst_bytes": "166",
        "dst_packets": "1",
        "start_time": "2024:08:29 2:14:38.861",
        "end_time": "2024:08:29 12:14:38.862",
        "intf_name": "0",
        "egress_intf_id": "0",
        "sys_up_time_first": "354805",
        "sys_up_time_last": "354806",
        "end_reason": "2",
        "app_name": "dns",
        "id": "3844000450554626049",
        "seq_num": "18085532",
        "ddsource": "gigamon"
      }
    result: null
  - sample: >-
      {
        "ts": "Thu Aug 29 12:15:37 2024",
        "vendor": "Gigamon",
        "version": "6.6.00",
        "generator": "gs_apps_appInst9_423330e5-4a3d-f203-14cb-b6642e5054e9",
        "dst_mac": "00:00:0c:9f:f7:f8",
        "src_mac": "00:50:56:b3:a4:e1",
        "src_ip": "10.203.43.1",
        "dst_ip": "91.189.91.49",
        "protocol": "6",
        "src_port": "55856",
        "dst_port": "80",
        "device_inbound_interface": "0",
        "http_server": "connectivity-check.ubuntu.com",
        "http_uri": "\\/",
        "http_uri_full": "\\/",
        "http_server_agent": "nginx\\/1.14.0 (Ubuntu)",
        "http_code": "204",
        "http_uri_path": "\\/",
        "http_request_size": "87",
        "http_host": "connectivity-check.ubuntu.com",
        "http_uri_decoded": "\\/",
        "http_uri_path_decoded": "\\/",
        "http_user_agent": "Mozilla\\/5.0 (Windows; U; Windows NT 6.1; en-US) AppleWebKit\\/534. 10 (KHTML, like Gecko) Chrome\\/8.0.552.237 Safari\\/534.10",
        "http_uri_raw": "\\/",
        "http_method": "GET",
        "http_version": "1.1",
        "app_id": "3006",
        "ip_version": "4",
        "src_bytes": "491",
        "dst_bytes": "461",
        "src_packets": "6",
        "dst_packets": "4",
        "start_time": "2024:08:29 12:15:36.202",
        "end_time": "2024:08:29 12:15:36.997",
        "intf_name": "0",
        "egress_intf_id": "0",
        "sys_up_time_first": "412146",
        "sys_up_time_last": "412941",
        "end_reason": "3",
        "app_name": "ubuntu",
        "id": "3844000451458498561",
        "seq_num": "18085527",
        "ddsource": "gigamon"
      }
    result: null
```

Reply:
Can you try committing this? I believe something is wrong with the file so want to see if this works